media-translation/cloud-client/requirements.txt

```
@@ -0,0 +1 @@
+google-cloud-media-translation>=0.1.0
```

```suggestion
google-cloud-media-translation==0.1.0
```
use the explicit version

---

Done.